code/game/dna/genes/powers.dm

```diff
@@ -115,23 +115,25 @@
 	activation_messages=list("Your muscles hurt.")
 	deactivation_messages=list("Your muscles shrink.")
 	instability = GENE_INSTABILITY_MAJOR
-	mutation=HULK
+	trait = TRAIT_HULK
 	activation_prob=15
 
 /datum/dna/gene/basic/hulk/New()
 	block=HULKBLOCK
 
-/datum/dna/gene/basic/hulk/activate(var/mob/M, var/connected, var/flags)
+/datum/dna/gene/basic/hulk/activate(mob/living/M, connected, flags)
 	..()
-	var/status = CANSTUN | CANWEAKEN | CANPARALYSE | CANPUSH
-	M.status_flags &= ~status
-
-/datum/dna/gene/basic/hulk/deactivate(var/mob/M, var/connected, var/flags)
+	M.status_flags &= ~CANPARALYSE
```

Comment:
Somewhere down the road we should consider why we have three / four way to say stun and are they necessary..